examples/meta/src/regression/linear_time_mmd.sg.in

```diff
@@ -0,0 +1,23 @@
+File f_feats_train = csv_file("@SHOGUN_DATA@/regression_1d_sinc_features_train.dat")
+File f_feats_test = csv_file("@SHOGUN_DATA@/regression_1d_sinc_features_test.dat")
+File f_labels_train = csv_file("@SHOGUN_DATA@/regression_1d_sinc_labels_train.dat")
+File f_labels_test = csv_file("@SHOGUN_DATA@/regression_1d_sinc_labels_test.dat")
+
+#![generate_data]
+#![generate_data]
+
+#![specify_kernels]
+#![specify_kernels]
+
+#![train_test_mode]
+#![train_test_mode]
+
+#![extract_result]
+#![extract_result]
+
+#![convex_combination]
+#![convex_combination]
+
+#![test]
+#![test]

```

Comment:
I am guessing you are still going to add the code here?

Reply:
Yes, I am! Do you have any resources for converting pseudocode/code from other languages to the sg format?